const_linux.go

```diff
@@ -0,0 +1,4 @@
+package main
+
+// MaxCommandBytes is the maximum number of bytes supported by exec
+const MaxCommandBytes = 2097152
```

Comment:
I guess it wouldn't actually be that slow in most cases, because the each process should run a bit faster.  I'll try with 32k.